app/ui/browser/views/browser.jsx

```diff
@@ -167,40 +168,22 @@ export default BrowserWindow;
 function attachIPCRendererListeners(browserView) {
   const { dispatch, ipcRenderer } = browserView.props;
 
-  ipcRenderer.on('profile-diff', (event, args) => {
-    dispatch(args);
-  });
-
-  ipcRenderer.on('new-tab', () => dispatch(actions.createTab()));
-
-  // TODO: Avoid this re-dispatch back to the main process
-  ipcRenderer.on('new-window', () => {
-    ipcRenderer.send('new-window');
-  });
-
-  ipcRenderer.on('show-bookmarks', () => {
-    dispatch(actions.createTab('atom://bookmarks'));
-  });
-
-  ipcRenderer.on('open-bookmark', (e, bookmark) => {
-    dispatch(actions.createTab(bookmark.url));
-  });
-
-  ipcRenderer.on('tab-attach', (e, tabInfo) => {
-    const page = tabInfo.page;
-    page.guestInstanceId = tabInfo.guestInstanceId;
-    dispatch(actions.attachTab(page));
-  });
+  ipcRenderer.on('profile-diff', (_, args) => dispatch(args));
+  ipcRenderer.on('focus-url-bar', () => dispatch(ipcActions.focusURLBar()));
```

Comment:
Is this working? Cmd-L isn't focusing the url bar for me. (it wasn't before either)